src/main/scala/sbttravisci/TravisCiPlugin.scala

```diff
@@ -3,9 +3,26 @@ package sbttravisci
 import sbt._, Keys._
 
 object TravisCiPlugin extends AutoPlugin {
+  private val DefaultScalaVersion = "2.10.6"    // for compatibility with stock sbt
```

Comment:
Should we default to 2.12.0 instead?  I just set it to 2.10 since that's the default SBT behavior if you don't specify `scalaVersion`.